examples/express-session-parse/index.js

```diff
@@ -52,13 +52,17 @@ app.delete('/logout', function (request, response) {
 // Create HTTP server by ourselves.
 //
 const server = http.createServer(app);
-const wss = new WebSocket.Server({ clientTracking: false, noServer: true });
+const wss = new WebSocket.Server({
+  clientTracking: false,
+  noServer: true // Manually handle upgrade
```

Comment:
I'm not very happy with this. I find it inconsistent. There is a comment only for the `noServer` option. There should a comment for every option or no comment.

I would prefer the user to refer to https://github.com/websockets/ws/blob/master/doc/ws.md#new-websocketserveroptions-callback.